src/components/accounts/accounts-component/accounts-component.js

```diff
@@ -0,0 +1,93 @@
+import template from "./accounts-component.html";
+import accountItemTemplate from "./account-item.html";
+import { ButtonMoreComponent } from "../../button-more/button-more";
+import { AddAccountComponent } from "../add-account-dialog/add-account-dialog";
+import { AccountService } from "../../../services/account-service";
+
+export class AccountsComponent {
+  constructor(mountPoint, props) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  querySelectors() {
+    this.accountsRoot = this.mountPoint.querySelector(".accounts");
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
+      ".account__more-button"
+    );
+    this.addAccountButton = this.mountPoint.querySelector(
+      ".account__add-account-dialog-activation"
+    );
+    this.accList = this.mountPoint.querySelector(".account__list");
+  }
+
+  initDialogComponent() {
+    this.addAccountComponent = new AddAccountComponent(
+      this.props.addAccountMountPoint,
+      {
+        onAddAccountConfirmed: this.handleAddAccountConfirmed.bind(this)
+      }
+    );
+  }
+
+  handleAddAccountConfirmed(account) {
+    AccountService.get().then(accounts => {
```

Comment:
please in `AccountService` create one more method called `add` -- and move this logic there